react-native-plaid-link-sdk.d.ts

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { TouchableOpacity, TouchableOpacityProps } from 'react-native';
+
+export interface OnSuccessMetadata {
+  status: string;
+  link_session_id: string;
+  request_id: string;
+  public_token: string;
+  account_id: string;
+  institution: {
+    institution_id: string;
+    name: string;
+  };
+  accounts: Array<{
+    id: string;
+    name: string;
+    mask: string;
+    type: string;
+    subtype: string;
+    verification_status?: {
+      pending_automatic_verification: boolean;
+      pending_manual_verification: boolean;
+      manually_verified: boolean;
+    };
+  }>;
+  account: {
+    id: string;
+    mask: string;
+    name: string;
+    subtype: string;
+    type: string;
+  };
```

Comment:
I think it might be a good idea to export it as a separate interface. In that case we will be able to use it in `accounts` array also (extending with verification_status).

Reply:
Thanks for the tip, I've incorporated the changes.